project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject mvxcvi/clj-pgp "1.1.1"
+(defproject mvxcvi/clj-pgp "1.1.2"
```

Comment:
In general, you probably shouldn't bump versions of other people's open source projects. 😛 It's up to the maintainer when to release and what version number to use next. I'm okay with it this time, since this code will no longer match what was released in 1.1.1, but it should also include a `-SNAPSHOT` suffix to indicate that this is not the final code for this version.
```suggestion
(defproject mvxcvi/clj-pgp "1.1.2-SNAPSHOT"
```